framework/source/class/qx/ui/core/queue/Visibility.js

```diff
@@ -41,8 +44,11 @@ qx.Class.define("qx.ui.core.queue.Visibility",
      */
     remove : function(widget)
     {
-      delete this.__data[widget.$$hash];
```

Comment:
This seems to be the cause of the failing tests (at least on my local linux karma tests). It seems that the this__lookup and this.__data hash is not allways in sync.

To fix this I've moved the ```delete this.__data[widget.$$hash];``` call out of the if statement again.